stetho/src/main/java/com/facebook/stetho/inspector/elements/android/RealWindowDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.view.View;
+
+import com.facebook.stetho.common.Accumulator;
+import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.android.window.WindowFetcher;
+import com.facebook.stetho.inspector.elements.android.window.WindowWrapper;
+
+import java.util.List;
+
+import static com.facebook.stetho.inspector.elements.android.ApplicationDescriptor.belongToActivity;
+
+
+public class RealWindowDescriptor extends AbstractChainedDescriptor<WindowWrapper> {
+	static {
+		WindowFetcher.hook();
+	}
+
+	@Override
+	protected void onGetChildren(WindowWrapper element, Accumulator<Object> children) {
+		List<View> windows = WindowFetcher.getWindowViews(element.getView().getContext());
```

Comment:
This code block seems to be duplicated exactly as-is from ApplicationDescriptor; why?  Can it be re-used?  Why is the logic even needed to run in two separate contexts?

Reply:
this class is useless, I have deleted